docs/sources/mimir/send/native-histograms/_index.md

```diff
@@ -172,46 +186,91 @@ Use the latest version of [Grafana Alloy](https://grafana.com/docs/alloy/<ALLOY_
+          Buckets: prometheus.DefBuckets,  // If buckets weren't already defined.
+          NativeHistogramBucketFactor: 1.1,
+          NativeHistogramMaxBucketNumber: 100,
+          NativeHistogramMinResetDuration: 1*time.Hour,
+   })
+   ```
+
+   ```java
+   static final Histogram requestLatency = Histogram.build()
+      .name("requests_latency_seconds")
+      .help("Histogram of request latency in seconds")
+      .classicUpperBounds(Histogram.Builder.DEFAULT_CLASSIC_UPPER_BOUNDS)  // If upper bounds weren't already defined.
+      .nativeInitialSchema(3)
+      .nativeMaxNumberOfBuckets(100)
+      .nativeResetDuration(1, TimeUnit.HOURS)
+      .register();
+   ```
+
+   {{< /code >}}
+
 1. Let Prometheus or Grafana Alloy scrape both classic and native histograms for metrics that have both defined.
-1. Send native histograms to remote write. Native histograms are sent to remote write by default if classic histograms are scraped.
-1. Start modifying the recording rules, alerts, and dashboards to use native histograms.
-1. After configuring native histogram collection, choose one of the following ways to stop collecting classic histograms.
+1. Send native histograms to remote write, along with the existing classic histograms.
+1. Modify dashboards to use the native histogram metrics. Refer to [Visualize native histograms](https://grafana.com/docs/mimir/<MIMIR_VERSION>/visualize/native-histograms/) for more information.
 
-   - Remove the custom bucket definition, `Buckets`/`classicUpperBounds`, from the instrumentation.
-   - Drop the classic histogram series with [Prometheus relabeling](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config) or [Grafana Alloy prometheus.relabel](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/prometheus/prometheus.relabel) at the time of scraping.
-   - Stop scraping the classic histogram version of metrics. This option applies to all metrics of a scrape target.
+   Use one of the following strategies to update dashbaords.
 
-Code examples with both classic and native histogram defined for the same metric:
+   - (Recommended) Add new dashboards with the new native histogram queries. This solution requires looking at different dashboards for data before and after the migration, until data before the migration is removed due to passing its retention time. Thus it is recommended to publish the new dashboard when sufficient time has passed to serve users with the new data.
+   - Add a dashboard variable to your dashboard to enable switching between classic histograms and native histograms. There isn't support for selectively enabling and disabling queries in Grafana yet ([issue 79848](https://github.com/grafana/grafana/issues/79848)). As a workaround, add the dashboard variable `latency_metrics`, for example, and assign it a value of either `-1` or `1`. Then, add the following two queries to the panel:
```

Comment:
```suggestion
   - Add a dashboard variable to your dashboard to enable switching between classic histograms and native histograms. There isn't support for selectively enabling and disabling queries in Grafana ([issue 79848](https://github.com/grafana/grafana/issues/79848)). As a workaround, add the dashboard variable `latency_metrics`, for example, and assign it a value of either `-1` or `1`. Then, add the following two queries to the panel:
```

Reply:
We try to keep docs always current, without referencing support that may be added in the future.